lib/Producers/MixpanelGroups.php

```diff
@@ -0,0 +1,103 @@
+        );
+        if ($ignore_time === true) $payload['$ignore_time'] = true;
+        return $payload;
+    }
+
+    /**
+     * Set properties on a group profile. If the group profile does not exist, it creates it with these properties.
+     * If it does exist, it sets the properties to these values, overwriting existing values.
+     * @param string|int $group_key the group_key used for groups in Project Settings
+     * @param string|int $group_id the group id used for the group profile
+     * @param array $props associative array of properties to set on the group profile
+     * @param boolean $ignore_time If the $ignore_time property is true, Mixpanel will not automatically update the "Last Seen" property of the group profile. Otherwise, Mixpanel will add a "Last Seen" property associated with the current time
+     */
+    public function set($group_key, $group_id, $props, $ignore_time = false) {
+        $payload = $this->_constructPayload($group_key, $group_id, '$set', $props, $ignore_time);
+        $this->enqueue($payload);
+    }
+
+    /**
+     * Set properties on a group profile. If the Group profile does not exist, it creates it with these properties.
+     * If it does exist, it sets the properties to these values but WILL NOT overwrite existing values.
+     * @param string|int $group_key the group_key used for groups in Project Settings
+     * @param string|int $group_id the group id used for the group profile
+     * @param array $props associative array of properties to set on the group profile
+     * @param boolean $ignore_time If the $ignore_time property is true, Mixpanel will not automatically update the "Last Seen" property of the group profile. Otherwise, Mixpanel will add a "Last Seen" property associated with the current time
+     */
+    public function setOnce($group_key, $group_id, $props, $ignore_time = false) {
+        $payload = $this->_constructPayload($group_key, $group_id, '$set_once', $props, $ignore_time);
+        $this->enqueue($payload);
+    }
+
+    /**
+     * Unset properties on a group profile. If the group does not exist, it creates it with no properties.
+     * If it does exist, it unsets these properties. NOTE: In other libraries we use 'unset' which is
+     * a reserved word in PHP.
+     * @param string|int $group_key the group_key used for groups in Project Settings
+     * @param string|int $group_id the group id used for the group profile
+     * @param array $props associative array of properties to unset on the group profile
+     * @param boolean $ignore_time If the $ignore_time property is true, Mixpanel will not automatically update the "Last Seen" property of the group profile. Otherwise, Mixpanel will add a "Last Seen" property associated with the current time
+     */
+    public function remove($group_key, $group_id, $props, $ignore_time = false) {
+        $payload = $this->_constructPayload($group_key, $group_id, '$unset', $props, $ignore_time);
```

Comment:
good catch, @zihejia 
```suggestion
        $payload = $this->_constructPayload($group_key, $group_id, '$remove', $props, $ignore_time);
```